### DISCUSSION TITLE
part-ii/2018-85/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#347] [Question comments]
Part II 2018 Linear Analysis: Paper 4, Section II, F 

https://questions.tripos.org/part-ii/2018-85/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

Part (c) seems to be unrelated to part (b).

Equip $C\left(S^1\right)$ with the $\infty$ norm, then take $$\mu(f) = \lim_{n\to\infty} \frac{\sum_{i=0}^{n-1}T^i(f(1))}{n}$$. 

</details>
